target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -69,6 +69,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	dwr-512-b|\
```

Comment:
if ```swconfig dev switch0 show``` shows the mapping you are looking for, yes. But please test the change afterwards. I might be wrong or have done something wrong.